### ISSUE TITLE
RDPC-Gateway callable restart task in jenkins

### ISSUE THREAD

**Issue by andricDu** [icgc-argo/workflow-roadmap#24]
When song-search or workflow-search get updated, it should trigger a task to restart rdpc-gateway so it can slurp up the new schemas.

Ideally if it fails to start we should report a failure in Jenkins. This means helm will not be sufficient as the only tool to achieve this.

**Comment by andricDu** [comment]
### Summary

- Task created and tested
- Added to jenkinsfile for song-search and tested
- Added to jenkinsfile for workflow-search and tested

### Note

Current implementation is using a half minute timeout after the helm upgrade for either downstream service. While this works, it is technically not a sound solution as it is inherently a race. However this is sufficient for dev/qa environments. 

**TL;DR DO NOT USE THIS SOLUTION FOR PROD**
